lib/Elevate/Components/MySQL.pm

```diff
@@ -360,6 +362,41 @@ sub _blocker_mysql_upgrade_in_progress ($self) {
     return 0;
 }
 
+sub _blocker_mysql_database_corrupted ($self) {
+
+    # We cannot run mysqlcheck if mysql is not running
+    if ( !Cpanel::MysqlUtils::Running::is_mysql_running() ) {
+
+        if ( Cpanel::Services::Enabled::is_enabled('mysql') ) {
+            my $db_version = Elevate::Database::get_local_database_version();
+            my $db_type    = Elevate::Database::get_database_type_name_from_version($db_version);
+
+            WARN("$db_type is not running; this prevents us from verifying the integrity of the databases.");
```

Comment:
FIX: Instead of throwing a warning and returning, we should attempt to restart the database server here and block if it fails to start.

Reply:
Okay, can do.